### DISCUSSION TITLE
Auto stop "Asus Services Running" function

### DISCUSSION THREAD

**Discussion by BlueCricket971** [seerge/g-helper#2457] [Ideas]
Every now an then I notice that all the Asus services are running again. But I only notice it if I go into the "extra" setting page and see if they are running or not. It makes sense that the Asus services start up every now and then, but sometimes I'll never know unless I frequently go into the extra panel to check. So, could there be a toggle option to auto stop Asus services? So when they start up again by themselves, G-helper automatically stops them from running.

**Comment by Kikimahe** [comment]
Hi,

ASUS services will remain disabled, **except** if you install an update ASUS driver.
They will be activated automatically by the ASUS driver installation file.
